src/cli.rs

```diff
@@ -94,11 +94,11 @@ fn load_plugin(path: &std::path::Path, context: &mut Context) -> Result<(), Shel
                 },
                 Err(e) => {
                     trace!("incompatible plugin {:?}", input);
-                    Err(ShellError::string(format!("Error: {:?}", e)))
+                    Err(ShellError::labeled_error(format!("Error: {:?}", e)))
```

Comment:
`ShellError::string` and `ShellError::labeled_error` work differently, so you'll need more than a rename here.

What labeled_error does is to give the error more information. Rather than just a short message, it will also underline part of the original source and put a label on it.  For that, it needs more parameters to be passed to the function.

You should look for uses of labeled_error in other parts of the code to see how it works.

I'd also recommend building Nu once you've made the changes and trying out the errors that you've modified to see how they look.